native/cocos/bindings/manual/jsb_ar_manual.cpp

```diff
@@ -0,0 +1,480 @@
+
+    const auto& args = s.args();
+    size_t argc = args.size();
+    if (argc == 0) {
+        float* buffer = cobj->getUpdatedSceneMesh();
+        int len = cobj->getInfoLength();
+        se::Object* planesInfo = se::Object::createTypedArray(se::Object::TypedArrayType::FLOAT32, buffer, sizeof(float) * len);
+        s.rval().setObject(planesInfo);
+        return true;
+    }
+
+    SE_REPORT_ERROR("wrong number of arguments: %d, was expecting %d", (int)argc, 0);
+    return false;
+}
+SE_BIND_FUNC(js_ar_ARModule_getUpdatedSceneMesh)
+
+static bool js_ar_ARModule_getRemovedSceneMesh(se::State& s)
+{
+    cc::ar::ARModule* cobj = (cc::ar::ARModule*)s.nativeThisObject();
+    SE_PRECONDITION2(cobj, false, "js_ar_ARModule_getRemovedSceneMesh : Invalid Native Object");
+
+    const auto& args = s.args();
+    size_t argc = args.size();
+    if (argc == 0) {
+        int* buffer = cobj->getRemovedSceneMesh();
+        int len = cobj->getInfoLength();
+        se::Object* meshesInfo = se::Object::createTypedArray(se::Object::TypedArrayType::UINT32, buffer, sizeof(float) * len);
+        s.rval().setObject(meshesInfo);
+        return true;
+    }
+
+    SE_REPORT_ERROR("wrong number of arguments: %d, was expecting %d", (int)argc, 0);
+    return false;
+}
+SE_BIND_FUNC(js_ar_ARModule_getRemovedSceneMesh)
+
+static bool js_ar_ARModule_requireSceneMesh(se::State& s)
+{
+    cc::ar::ARModule* cobj = (cc::ar::ARModule*)s.nativeThisObject();
+    SE_PRECONDITION2(cobj, false, "js_ar_ARModule_requireSceneMesh : Invalid Native Object");
+
+    const auto& args = s.args();
+    size_t argc = args.size();
+    if (argc == 0) {
+        int* buffer = cobj->requireSceneMesh();
+        int len = cobj->getInfoLength();
+        se::Object* meshesInfo = se::Object::createTypedArray(se::Object::TypedArrayType::UINT32, buffer, sizeof(float) * len);
+        s.rval().setObject(meshesInfo);
+        return true;
+    }
+
+    SE_REPORT_ERROR("wrong number of arguments: %d, was expecting %d", (int)argc, 0);
+    return false;
+}
+SE_BIND_FUNC(js_ar_ARModule_requireSceneMesh)
+
+static bool js_ar_ARModule_getSceneMeshVertices(se::State& s)
+{
+    cc::ar::ARModule* cobj = (cc::ar::ARModule*)s.nativeThisObject();
+    SE_PRECONDITION2(cobj, false, "js_ar_ARModule_getSceneMeshVertices : Invalid Native Object");
+
+    const auto& args = s.args();
+    size_t argc = args.size();
+    CC_UNUSED bool ok = true;
+    if (argc == 1) {
+        HolderType<int, false> arg0 = {};
+        ok &= sevalue_to_native(args[0], &arg0, s.thisObject());
+        SE_PRECONDITION2(ok, false, "js_ar_ARModule_getSceneMeshVertices : Error processing arguments");
+
+        float* buffer = cobj->getSceneMeshVertices(arg0.value());
+        int len = cobj->getInfoLength();
+        se::Object* verticesInfo = se::Object::createTypedArray(se::Object::TypedArrayType::FLOAT32, buffer, sizeof(float) * len);
```

Comment:
se::Object is leak here.
Pure JS object ( not a JSB object) should use `se::HandleObject` to ensure the pure object being deallocated.
```suggestion
        se::HandleObject verticesInfo{se::Object::createTypedArray(se::Object::TypedArrayType::FLOAT32, buffer, sizeof(float) * len)};
```